miio/airfresh_t2017.py

```diff
@@ -397,3 +434,88 @@ def get_ptc_timer(self):
     def get_timer(self):
         """Response unknown."""
         return self.send("get_timer")
+
+
+class AirFreshT2017(AirFreshA1):
+    """Main class representing the air fresh t2017."""
+
+    def __init__(
+        self,
+        ip: str = None,
+        token: str = None,
+        start_id: int = 0,
+        debug: int = 0,
+        lazy_discover: bool = True,
+        model: str = MODEL_AIRFRESH_T2017,
+    ) -> None:
+        super().__init__(ip, token, start_id, debug, lazy_discover)
+
+        if model in AVAILABLE_PROPERTIES:
+            self.model = model
+        else:
+            self.model = MODEL_AIRFRESH_T2017
+
+    @command(
+        default_output=format_output(
+            "",
+            "Power: {result.power}\n"
+            "Mode: {result.mode}\n"
+            "PM2.5: {result.pm25}\n"
+            "CO2: {result.co2}\n"
+            "Temperature: {result.temperature}\n"
+            "Favorite speed: {result.favorite_speed}\n"
+            "Control speed: {result.control_speed}\n"
+            "Dust filter life: {result.dust_filter_life_remaining} %, "
+            "{result.dust_filter_life_remaining_days} days\n"
+            "Upper filter life remaining: {result.upper_filter_life_remaining} %, "
+            "{result.upper_filter_life_remaining_days} days\n"
+            "PTC: {result.ptc}\n"
+            "PTC level: {result.ptc_level}\n"
+            "PTC status: {result.ptc_status}\n"
+            "Child lock: {result.child_lock}\n"
+            "Buzzer: {result.buzzer}\n"
+            "Display: {result.display}\n"
+            "Display orientation: {result.display_orientation}\n",
+        )
+    )
+    def status(self) -> AirFreshStatus:
+        """Retrieve properties."""
+
+        return super().status()
```

Comment:
```suggestion
```
This could be removed as no-op, right?

Reply:
I added this no-op method because the @command output is slightly changed. I think

Reply:
Do you agree? :-)

Reply:
Oh, I see, I think that's fine then :-)

Generally speaking, I think we should start thinking how to move away from hard-coded responses (maybe simply using the status classes' property markings?) to make it less error-prone and easier to maintain.. Considering that will affect the whole code-base, it is not the task for this PR, just something to keep in mind :-)